masq/src/commands/financials_command.rs

```diff
@@ -4,52 +4,155 @@ use crate::command_context::CommandContext;
 use crate::commands::commands_common::{
     dump_parameter_line, transaction, Command, CommandError, STANDARD_COMMAND_TIMEOUT_MILLIS,
 };
-use clap::{App, SubCommand};
-use masq_lib::messages::{UiFinancialsRequest, UiFinancialsResponse};
+use clap::{App, Arg, ArgGroup, ArgMatches, SubCommand};
+use masq_lib::messages::{
+    CustomQueries, QueryResults, RangeQuery, TopRecordsConfig, UiFinancialStatistics,
+    UiFinancialsRequest, UiFinancialsResponse, UiPayableAccount, UiReceivableAccount,
+};
+use masq_lib::shared_schema::common_validators::validate_non_zero_u16;
 use masq_lib::short_writeln;
-use std::fmt::Debug;
+use masq_lib::utils::{plus, ExpectValue};
+use num::CheckedMul;
+use regex::{Captures, Regex};
+use std::cell::RefCell;
+use std::collections::VecDeque;
+use std::default::Default;
+use std::fmt::{Debug, Display};
+use std::io::Write;
+use std::num::{IntErrorKind, ParseIntError};
+use std::ops::{Add, Mul};
+use std::str::FromStr;
+use thousands::Separable;
 
 const FINANCIALS_SUBCOMMAND_ABOUT: &str =
     "Displays financial statistics of this Node. Only valid if Node is already running.";
+const TOP_ARG_HELP: &str = "Returns a subset of the top N records (or fewer, if their total count is smaller) from both payable and receivable. By default, the ordering is done by balance but can be changed with the additional '--ordered' argument";
+const PAYABLE_ARG_HELP: &str = "Forms a detailed query about payable records by specifying two ranges, one for their age in seconds and another for their balance in MASQs (decimal numbers are supported, allowing Gwei precision). The desirable format of those values is <MIN-AGE>-<MAX-AGE>|<MIN-BALANCE>-<MAX-BALANCE>";
```

Comment:
The | in the instructions looks like a metacharacter that says, "You can type either this or that." Make sure the reader knows that here it's a real character that you actually have to type, or it could be confusing.

Suggestion: if you made it a & instead of a |, it might make more semantic sense. However, if you do that you're going to have trouble with noninteractive mode, because & means something to the shell. Oh well...